x-pack/test/performance/scalability/config.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrConfigProviderContext } from '@kbn/test';
+import fs from 'fs';
+import path from 'path';
+import { serializeApmGlobalLabels } from '../utils';
+import { ScalabilityTestRunner } from './runner';
+
+// These "secret" values are intentionally written in the source.
+const APM_SERVER_URL = 'https://142fea2d3047486e925eb8b223559cae.apm.europe-west1.gcp.cloud.es.io';
+const APM_PUBLIC_TOKEN = 'pWFFEym07AKBBhUE2i';
+const AGGS_SHARD_DELAY = process.env.LOAD_TESTING_SHARD_DELAY;
+const DISABLE_PLUGINS = process.env.LOAD_TESTING_DISABLE_PLUGINS;
+
+export default async function ({ readConfigFile, log }: FtrConfigProviderContext) {
+  const performanceConfig = await readConfigFile(require.resolve('../journeys/base.config.ts'));
+
+  const scalabilityJsonPath = process.env.SCALABILITY_JOURNEY_PATH;
+  const isJsonPathDefined =
+    scalabilityJsonPath &&
+    fs.existsSync(scalabilityJsonPath) &&
+    path.extname(scalabilityJsonPath) === '.json';
+
+  const apmGlobalLabels = {
+    ...performanceConfig.get('kbnTestServer').env.ELASTIC_APM_GLOBAL_LABELS,
+    journeyFilePath: isJsonPathDefined ? path.basename(scalabilityJsonPath) : 'unknown',
+    journeyName: isJsonPathDefined
```

Comment:
Should we treat this as a hard error if we don't have a valid JSON file at this point?